spec/rabbitmq_spec.rb

```diff
@@ -0,0 +1,17 @@
+
+require 'spec_helper'
+
+
+describe RabbitMQ do
+  describe "DEFAULT_EXCHANGE" do
+    subject { RabbitMQ::DEFAULT_EXCHANGE }
+    
+    it { should eq "" }
+  end
+
+  describe "AVAILABILITY_ERRORS" do
+    subject { RabbitMQ::AVAILABILITY_ERRORS }
+
+    it { should be_an Array }
```

Comment:
Maybe this could also test that the array is non-empty, and that each element is a subclass of `Exception`?